public/components/agents/prompts/prompt_module_not_for_agent.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Wazuh app - Component for the module generate reports
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { EuiButton, EuiEmptyPrompt } from '@elastic/eui';
+import { updateCurrentAgentData } from '../../../redux/actions/appStateActions';
+import { useFilterManager } from '../../common/hooks';
+
+type PromptSelectAgentProps = {
+  body?: string;
+  title: string;
+  agentSelectionProps: {
+    setAgent: (agent: boolean) => void
+  }
+};
+
+export const PromptModuleNotForAgent = ({ body, title, ...agentSelectionProps }: PromptSelectAgentProps) => {
+  const dispatch = useDispatch();
+  const filterManager = useFilterManager();
```

Comment:
This has changed in the office365 branch. Noting it so we are mindful when merging there.